portal-impl/src/com/liferay/portal/service/impl/UserGroupLocalServiceImpl.java

```diff
@@ -181,6 +186,10 @@ public boolean addGroupUserGroups(long groupId, long[] userGroupIds) {
 
 		try {
 			reindexUsers(userGroupIds);
+
+			for (long userGroupId : userGroupIds) {
+				reindexUserGroup(getUserGroup(userGroupId));
```

Comment:
I think that we should reindex user groups when we are calling to `unsetGroupUserGroups` and also when we delete a groupId, for this use case would be when we are deleting an space (asset library ) .

Reply:
Hi @robertoDiaz should we add the reindexing also when deleting a groupId (asset library) ?